apps/services/sessions/sequelize.config.js

```diff
@@ -1,12 +1,12 @@
 /* eslint-env node */
 module.exports = {
   development: {
-    username: 'dev_db',
-    password: 'dev_db',
-    database: 'dev_db',
+    username: 'dev_db' ?? process.env.DB_USER_SESSIONS,
+    password: 'dev_db' ?? process.env.DB_PASS_SESSIONS,
+    database: 'dev_db' ?? process.env.DB_NAME_SESSIONS,
```

Comment:
intentional or forgot to remove ?

Reply:
Intentional to help with DX. But this has to be the other way around so I fixed it 😅